Projects/Claims/Sources/Models/ClaimModel.swift

```diff
@@ -1,203 +1,131 @@
 import Foundation
+import hCore
 import hGraphQL
 
-typealias ClaimStatusCard = GiraffeGraphQL.ClaimStatusCardsQuery.Data.ClaimsStatusCard
-
 public struct ClaimModel: Codable, Equatable, Identifiable, Hashable {
+
     public init(
         id: String,
-        pills: [ClaimModel.ClaimPill],
-        segments: [ClaimModel.ClaimStatusProgressSegment],
-        title: String,
-        subtitle: String,
-        claimDetailData: ClaimDetailData
+        status: ClaimStatus,
+        outcome: ClaimOutcome,
+        submittedAt: String?,
+        closedAt: String?,
+        signedAudioURL: String,
+        statusParagraph: String,
```

Comment:
dont need this in init -> statusParagraph

Reply:
will remove this